lite2/client.go

```diff
@@ -630,13 +630,14 @@ func (c *Client) Witnesses() []provider.Provider {
 // client must be stopped at this point.
 func (c *Client) Cleanup() error {
 	c.logger.Info("Removing all the data")
-	return c.cleanup(0)
+	return c.cleanup(0, 0)
```

Comment:
we can now use `trustedStore.Prune(0)` function

Reply:
Yup we could